packages/analytics-js/src/components/configManager/ConfigManager.ts

```diff
@@ -36,26 +36,26 @@ class ConfigManager implements IConfigManager {
    */
   init() {
     validateLoadArgs(state.lifecycle.writeKey.value, state.lifecycle.dataPlaneUrl.value);
-    // determine the sourceConfig url
+    const lockIntegrationsVersion = state.loadOptions.value.lockIntegrationsVersion || false;
```

Comment:
```suggestion
    const lockIntegrationsVersion = state.loadOptions.value.lockIntegrationsVersion === true;
```

Reply:
why do we want to lock the version by default?

Reply:
It's quite the opposite. Read again.
We're assigning `true` only if the incoming option is `true` and everything else resolves to `false`.

Reply:
Got it